EPRN.Portal/Helpers/HomeServiceFactory.cs

```diff
@@ -0,0 +1,40 @@
+﻿using EPRN.Common.Enums;
+using EPRN.Portal.Helpers.Interfaces;
+using EPRN.Portal.Services.Interfaces;
+
+namespace EPRN.Portal.Helpers
+{
+    public class HomeServiceFactory : IHomeServiceFactory
+    {
+        private readonly IHomeService exporterHomeService;
+        private readonly IHomeService reprocessorHomeService;
+        private readonly IHomeService exporterAndReprocessorHomeService;
+        private readonly IWasteService wasteService;
+        private readonly IUserRoleService userRoleService;
+
+        public HomeServiceFactory(IHomeService exporterHomeService, IHomeService reprocessorHomeService, IHomeService exporterAndReprocessorHomeService, 
+            IWasteService wasteService, IUserRoleService userRoleService)
```

Comment:
Just a presentational thing: having lots of parameters on the same line makes reading of them difficult. When this happens it's best to put each one on a new line so that it's more maintainable, and more presentable.